.github/workflows/update-release-metadata.yml

```diff
@@ -0,0 +1,37 @@
+name: Update Release Metadata
+
+on:
+  workflow_dispatch:
+    inputs:
+      newVersion:
+        description: version override
+        type: string
+        required: false
+
+jobs:
+  create-update-pr:
+    name: Create update PR
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v4
```

Comment:
misread this as "set up node at version 4" for a second and was like "wow I knew the tech stack here was out of date but I didn't think it was that old"